index.html

```diff
@@ -7,99 +7,93 @@
-        <span><a href="#contact" class="menu-item">Contact</a></span>
-      </div>
+        <div class="wobble animated logo"><a href="#">My Logo</a></div>
+        <div class="menu">
+            <section class="union-bar">
+                <nav data-bss-hover-animate="pulse"><img src="/Assets/images/Union.png" alt="union-bar"></nav>
+            </section>
+        </div>
+        <div class="menu-toggle">
+          <span data-bss-hover-animate="rubberBand"><a href="#portfolio" class="menu-item">Portfolio</a></span>
+          <span data-bss-hover-animate="rubberBand"><a href="#about" class="menu-item">About</a></span>
+          <span data-bss-hover-animate="rubberBand"><a href="#contact" class="menu-item">Contact</a></span></div>
     </div>
-
-
   </div>
 
-
   <!-------main section------>
   <section class="headline">
     <section class="main" id="home">
-      <div class="intro-text">
-        <h1 class="intro-mobile">I’m Lucy. <br> Glad to see you!</h1>
-        <p class="text-description">I’m a software developer! I can help you build a product, feature or website Look
-          through some of my work and experience! If you like what you see and have a project you need coded, don’t
-          hestiate to contact me.</p>
-      </div>
-      <div class="social">
-        <h3>LET'S CONNECT</h3>
-        <ul class="media-icons">
-          <li><a><img src="Assets/images/Twitter.png" alt="twitter"></a></li>
-          <li><a><img src="Assets/images/Linkedin icon.png" alt="linkedin"></a></li>
-          <li><a><img src="Assets/images/Mail.png" alt="mail"></a></li>
-          <li><a><img src="Assets/images/Github.png" alt="github"></a></li>
-          <li><a><img src="Assets/images/peace.png" alt="vector"></a></li>
-        </ul>
-      </div>
+        <div class="intro-text">
+            <h1 data-bss-hover-animate="rubberBand" class="intro-mobile">I’m Lucy. <br> Glad to see you!</h1>
+            <p data-bss-hover-animate="rubberBand" class="text-description">I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </div>
+        <div class="social">
+            <h3 data-bss-hover-animate="flash">LET'S CONNECT</h3>
+            <ul class="media-icons">
+                <li data-bss-hover-animate="bounce"><a><img src="/Assets/images/Twitter.png" alt="twitter"></a></li>
+                <li data-bss-hover-animate="bounce"><a><img src="/Assets/images/Linkedin-icon.png" alt="linkedin"></a></li>
+                <li data-bss-hover-animate="bounce"><a><img src="Assets/images/Mail.png" alt="mail"></a></li>
+                <li data-bss-hover-animate="bounce"><a><img src="/Assets/images/Github.png" alt="github"></a></li>
+                <li data-bss-hover-animate="bounce"><a><img src="/Assets/images/peace.png" alt="peace"></a></li>
+            </ul>
```

Comment:
- [ ] Kindly consider adding a `.` in front of your image path as that will tell the `src` prop to look for the images in the current directory. Because you haven't specified where the `src` prop to fetch the photos from they are currently not loading in your deployed app. Please consider fixing this issue as it will help create a better user experience. See below for more details:
![IMG_0038](https://github.com/lucy-sees/Portfolio/assets/107192076/28d5c4db-3b40-4045-9a59-c55f0a3ca8ca)